Library/Format.swift

```diff
@@ -344,8 +372,8 @@ public enum Format {
 
 public let defaultThresholdInDays = 30 // days
 
-private struct DateFormatterConfig {
```

Comment:
Had to change these access rules to make `SwiftFormat` happy (otherwise the build would fail).